components/TextFields/tests/snapshot/MDCTextControlSnapshotTestHelpers.m

```diff
@@ -26,6 +30,27 @@ @implementation MDCTextControlSnapshotTestHelpers
 
 #pragma mark Validation
 
++ (void)addTextControlToKeyWindow:(UIView<MDCTextControl> *)textControl {
+  // Add the text control to a container view. This is the view we snapshot. Snapshotting this view
+  // instead of the text control allows us to see the floating label go outside the bounds for the
+  // outlined style.
+  UIView *container = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 1, 1)];
```

Comment:
why give it a frame so small?

Reply:
For some reason while working on it I was getting some error about zero width layers, so I made it non zero. But I just looked again, and apparently something has changed, and it's no longer an issue. So now I just initialize the view with CGRectZero.